pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/HelixBrokerStarter.java

```diff
@@ -82,16 +82,16 @@
   private ZkHelixPropertyStore<ZNRecord> _propertyStore;
   private HelixDataAccessor _helixDataAccessor;
 
-  // Cluster change handlers
-  private HelixExternalViewBasedRouting _helixExternalViewBasedRouting;
```

Comment:
It looks like, with these changes we can HelixExternalViewBasedRouting class and RoutingTable interface? (assuming RoutingManageris backward compatible)

Reply:
Yes. The code cleanup will be in the following pr.